android/src/main/java/com/appfolio/react/imageutils/ImageResizer.java

```diff
@@ -219,6 +229,39 @@ private static Bitmap loadBitmapFromBase64(@NonNull final Uri imageUri,
         return sourceImage;
     }
 
+    /**
+     * Load a Bitmap from an http or https URL.
+     */
+    @Nullable
+    public static Bitmap loadBitmapFromURL(@NonNull final Uri imageUri,
+                                           @NonNull final BitmapFactory.Options options) throws IOException {
+        Bitmap sourceImage = null;
+        URL url = null;
+
+        try {
+            url = new URL(imageUri.toString());
+        } catch (MalformedURLException e) {
+            throw new MalformedURLException("Unable to load image from a malformed URL: " + e.getMessage());
+        }
+
+        OkHttpClient client = new OkHttpClient();
```

Comment:
If possible, it would be nice to reuse the `OkHttpClient` from `OkHttpClientProvider` in react-native.

Reply:
Using `getOkHttpClient` from `OkHttpClientProvider`.